frontend/src/app/main/ui/hooks/resize.cljs

```diff
@@ -48,14 +49,23 @@
          start-size-ref (mf/use-ref nil)
          start-ref      (mf/use-ref nil)
 
-         window-height (dom/get-window-height)
+         window-height* (mf/use-state #(dom/get-window-height))
+         window-height (deref window-height*)
 
          max-val (mf/with-memo [max-val window-height]
                    (let [parsed-max-val (when (string? max-val) (d/parse-double max-val))]
                      (if parsed-max-val
                        (* window-height parsed-max-val)
```

Comment:
This will only work if the axis is `:y` (if you give an `:x` axis you will have a max-val depending on the window height.

This is ok for now, because currently we are not using percent max-val in any horizontal panel. But I will add, for example, a docstring to the hook explaining how a string max-val works and that it's only implemented for vertical axis. If later is needed to use this for a horizontal panel it won't be complicated to add it.